docs/cmdline-opts/aws-sigv4.d

```diff
@@ -8,3 +8,10 @@ Use AWS V4 signature authentication in the transfer.
 
 The provider argument is a string that is used by the algorithm when creating
 outgoing authentication headers.
+
+The region argument is a string that points to a geographic area of
+a resources collection (region-code) when the region name is omitted from
+the endpoint.
+
+The service argument is a string that points to a function provided by a cloud
+(service-code) when the service name is omitted from the endpoint.
```

Comment:
This confuses me. Are there always two providers if you specify a region? You can't specify one provider and service for example?

Reply:
@bagder, yep, you are right. It should specify two providers for using a region via opt.
But it can be specified one provider if region specified by URL endpoint. Unfortunately, there're no ways for omiting the second provider while the region is specified via opt.

See opt mask: `<provider1[:provider2[:region[:service]]]>`